src/components/anchored.module.scss

```diff
@@ -0,0 +1,28 @@
+.small {
+    font-weight: bold;
+}
+
+a.icon {
```

Comment:
This kind of selector is not supported by CSS modules classes reference, you will need to enclose it in the `.root` class to get it to work.